README.md

```diff
@@ -22,6 +30,17 @@ To create these collections:
 npm run db:setup
 ```
 
+### CRUD Operations
+
+This server currently supports the following CRUD operations:
+
+- Read by ID to endpoint: `4_0_1/<resourceType>/<resourceId>`
+  _More functionality coming soon!_
+
+### Search
+
+The Measure Repository Server supports `Library` and `Measure` resource search by the parameters specified in the shareable measure repository section of the [HL7 Measure Repository Docs](https://build.fhir.org/ig/HL7/cqf-measures/measure-repository-service.html#shareable-measure-repository).
```

Comment:
Do we want to list the parameters here so the user doesn't have to navigate to the docs? Might not be necessary I just saw that we do that in the test-server README.